Scheduler/observing_blocks_maintel/AOS/BLOCK-T81.json

```diff
@@ -0,0 +1,594 @@
+{
+    "name": "BLOCK-T81",
+    "program": "BLOCK-T81",
+    "constraints": [],
+    "configuration_schema": "$schema: http://json-schema.org/draft-07/schema#\ntitle: BLOCK-T81 Configuration\ndescription: Configuration for the BLOCK-T81 Reference Wavefront Generation Test WET-010.\ntype: object\nproperties:\n  ra:\n    description: Right Ascension.\n    type: number\n    default: 0\n  dec:\n    description: Declination.\n    type: number\n    default: 0\n  exposure_time:\n    description: Exposure Time.\n    type: number\n    default: 30.0\n",
+    "_schema_comment": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "BLOCK-T81 Configuration",
+    "description": "Configuration for the BLOCK-T81 Reference Wavefront Generation Test WET-010.",
+    "type": "object",
+    "properties": {
+      "ra": {
+        "description": "Right Ascension.",
+        "type": "number",
+        "default": 0
+      },
+      "dec": {
+        "description": "Declination.",
+        "type": "number",
+        "default": 0
+      },
+      "exposure_time": {
+        "description": "Exposure Time.",
+        "type": "number",
+        "default": "30.0"
+      }
+    }
+  },
+    "scripts": [
+        {
+            "name": "maintel/enable_comcam.py",
+            "standard": true,
+            "parameters": {}
+
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -82.8,
+                "rot_type": "PhysicalSky"
+            }
+        },    
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
```

Comment:
The default is already 1500, and since you are not making this configurable, we probably don't need this here.

Reply:
Resolved